apps/frontend/src/libs/components/select/libs/mappers/select-value.mapper.ts

```diff
@@ -0,0 +1,19 @@
+import { type SelectOption } from "../types/select-option.type.js";
+
+const mapSelectValue = <TOptionValue>(
+	rawValue: null | TOptionValue | TOptionValue[],
+	options: SelectOption<TOptionValue>[],
+	isMulti: boolean,
+): SelectOption<TOptionValue>[] => {
+	if (isMulti) {
+		return Array.isArray(rawValue)
+			? options.filter((option) => rawValue.includes(option.value))
+			: [];
+	}
+
+	const found = options.find((option) => option.value === rawValue);
```

Comment:
let's make it more clear
```suggestion
	const foundOption = options.find((option) => option.value === rawValue);
```